tests/test_taxi_service_home.py

```diff
@@ -29,6 +29,6 @@ def test_index_count_content_correctly(self):
 
 class IsStylesCSSExistTests(TestCase):
     def test_styles_exist(self):
-        file_exists = os.path.exists('static/css/styles.css')
+        file_exists = os.path.exists('static/css/stiles.css')
```

Comment:
do not change the test
it's better to rename the styles file

Reply:
It's happened automatically after refactoring name styles to styles. I thought styles is correct name